src/player.h

```diff
@@ -1219,7 +1219,7 @@ class Player final : public Creature, public Cylinder
 	Party* party = nullptr;
 	Player* tradePartner = nullptr;
 	SchedulerTask* walkTask = nullptr;
-	Town* town = nullptr;
+	const Town* town = nullptr;
```

Comment:
could you explain why it became a const?

Reply:
This is assigned with the return value of `getTown` (i.e. `player->town = towns->getTown(id)`) which became `const Town*`